src/components/editMeasure/details/MeasureMetaDataRow.tsx

```diff
@@ -2,18 +2,52 @@ import React from "react";
 // we will have two different renders so far
 // measureMetaData.measureDefinition = [{term, definition}];
 // measureMetaData.measureReferences = [{referenceText, ReferenceType}]
+import { MeasureDefinition } from "../../editMeasure/details/MeasureDefinitions/MeasureDefinitions";
+import { Button } from "@madie/madie-design-system/dist/react";
 
 interface MeasureMetaDataRowProps {
   name: string;
   description: string;
+  measureDefinition?: MeasureDefinition;
+  setOpen?: Function;
+  setSelectedDefinition?: Function;
 }
 
 const MeasureMetaDataRow = (props: MeasureMetaDataRowProps) => {
-  const { name, description } = props;
+  const {
+    name,
+    description,
+    measureDefinition,
+    setOpen,
+    setSelectedDefinition,
+  } = props;
+
+  const handleOpen = async (event: React.MouseEvent<HTMLButtonElement>) => {
+    setOpen(true);
+    setSelectedDefinition(measureDefinition);
+  };
```

Comment:
I think this should be maintained by the parent component. we'll have a similar callback for references as well in the future.

Reply:
@adongare setOpen() was also called in the parent component, but setSelectedDefinition() needs to be handled here because the button for edit is in the child component.

Reply:
@adongare Thanks so much Ashok!! This is refactored, I just pushed in the new changes.